src/main/java/org/spongepowered/api/util/command/CommandSource.java

```diff
@@ -44,6 +46,17 @@
     String getName();
 
     /**
+     * Gets the locale used by this command source. If this
+     * {@link CommandSource} does have a {@link Locale} configured or does not
+     * support configuring a {@link Locale}, the system default is used.
+     *
+     * @return The locale used by this command source
+     */
+    default Locale getLocale() {
+        return Locale.getDefault();
```

Comment:
oh, and:
> Our server should be stable enough to send proper/existing messages for any language (as long as at least one is provided)
The server only has `en_US` translations included, because everything send client-side (after the client-side locale is known, that is) uses translation keys and is translated client-side - console/etc messages use `en_US`. The exception for this is plugin-managed translations, which are done server-side.